### DISCUSSION TITLE
Java 18 Support on Hosted Agent

### DISCUSSION THREAD

**Discussion by ahsanmirxa** [actions/virtual-environments#6321] [Installed software]
When it is expected to add Java 18 on Microsoft Hosted Agent?

**Comment by al-cheb** [comment]
Hey @ahsanmirxa. 
We add only JAVA LTS version on our images. JAVA 18 is non-LTS version.

**Reply by ahsanmirxa** [reply]
makes sense, thanks @al-cheb
